core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -304,6 +332,10 @@ export class CkEditor4RteComponent implements AfterViewInit, OnDestroy {
           .css('width', '24px')
           .css('padding', '0px 0px');
       });
+      $('.cke_inner').append(
+        // eslint-disable-next-line max-len
```

Comment:
Please avoid skipping checks. This string can be broken into multiple lines.

Reply:
Also, here and below, I am not sure if we allow new jQuery methods to manipulate the DOM... Is there an alternative?

Reply:
Removed skipping checks comment. Also for the ckeditor we don't have pre defined component template so here we'll need to use jquery to disable using class name.